src/main/java/seedu/mtracker/commands/InvalidCommand.java

```diff
@@ -0,0 +1,10 @@
+package seedu.mtracker.commands;
+
+public class InvalidCommand extends Command {
+    public static final String COMMAND_WORD = null;
```

Comment:
Do rename it to "invalid"
```suggestion
    public static final String COMMAND_WORD = "invalid";
```